tools/cockpit.spec

```diff
@@ -634,7 +634,7 @@ bastion hosts, and a basic dashboard.
 %endif
 
 %ifarch x86_64 %{arm} aarch64 ppc64le i686 s390x
-%if 0%{?fedora}
+%if 0%{?fedora} && 0%{?fedora} <= 30
```

Comment:
Not sure why this is in the "docker: Skip docker tests on fedora-31" *and* the previous "add fedora-31" commit. (github web UI bug?). Either way, this commit should be squashed into the "add fedora-31 image" commit.